src/App/components/DisplayColor/index.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const ColorDisplay = ({ hex, rgb, fontColor, width, border }) => (
-    <div className={`${width ? width : ""}`}>
+const ColorDisplay = ({ hex, rgb, fontColor, border }) => (
+    <div className={`${border && border}`}>
```

Comment:
This gives the class `undefined` when there's no border prop. 
Perhaps it's better the same way the width prop was used. Like: 
```suggestion
    <div className={`${border ? border : ""}`}>
```